packages/cashier/src/Stores/Cashier/transaction-history-store.js

```diff
@@ -0,0 +1,30 @@
+import { action, observable } from 'mobx';
+
+export default class TransactionHistoryStore {
+    constructor({ WS }) {
+        this.WS = WS;
+    }
+
+    @observable crypto_transactions = [];
+
+    @action.bound setCryptoTransactionsHistory(transactions) {
+        this.crypto_transactions = transactions;
+    }
+
+    @action.bound
+    async getCryptoTransactions() {
+        await this.WS.send({ cashier_payments: 1, provider: 'crypto', transaction_type: 'all' }).then(response => {
```

Comment:
It's better to define this api call in `socket_base.js` in core and just call it from here.

Reply:
@Bahar-FS  I did it at first, but then found another pr that had the same API call but in store, is there a difference or socket-base.js is the file for all  API calls?

Reply:
@kseniya-deriv `socket_base.js` is the best place for defining api calls, although we can define them in our store, but please follow the best way.